src/main.rs

```diff
@@ -25,6 +27,19 @@ struct Opt {
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     let opt = Opt::from_args();
+
+    if opt.list {
+        println!("Supported Licenses are:");
+        use license_generator::license::LICENSES;
+        for l in LICENSES {
+            print!("{}, ", l);
+        }
+        println!();
+        return Ok(());
+    } else {
+        
+    }
```

Comment:
Is this `else` clause necessary?

Reply:
Ah, for goodness’ sake. I did fix this on my local repo, but git was not liking me yesterday. No, that else clause is very much not necessary.